session-name-1/activities/activity-folder-1/sol/solution.py

```diff
@@ -3,9 +3,17 @@
 
 PORT = 8001
 
-if len(sys.argv) > 1 and sys.argv[1] == 'local':
-    HOST = '127.0.0.1'
+if len(sys.argv) == 1:
+    print(f"Usage: {sys.argv[0]} [local|remote]\nor")
+    print(f"Usage: {sys.argv[0]} <ip> <port>")
```

Comment:
I think we should get rid of these hardcoded ports and only supply them via command-line args. So only the second usage should be accepted.

Reply:
Isn't this a more general approach tho?
At least for localhost we should keep it, it pretty simple to test without adding port and localhost.

Reply:
Alright. If it's for debugging it can stay.